### DISCUSSION TITLE
PID data doesn't match Torque PID (customer PID in use)

### DISCUSSION THREAD

**Discussion by rjterry21** [agronick/aa-torque#90] [General]
I've noticed the displayed values in aa-torque for the custom PID's I have in Torque-Pro (Cummins diesel) do not match the Torque PID values as shown in Torque. Do I need to input all the formulas that exist in the Torque custom PID's into the PID selected in aa-torque?

**Comment by agronick** [comment]
Yes Torque sends over the unconverted values. You need to convert them into whatever format you want them.
